web/src/components/Storyblok/PromoBox/PromoBox.tsx

```diff
@@ -61,9 +66,20 @@ export const PromoBox: React.FC<PromoBoxProps> = ({
 		? constructStoryblokImageSrc(image?.filename)
 		: undefined;
 
+	const verticalPaddingClass = `promoBox${toTitleCase(verticalPadding)}Spacing`;
+
+	const imageAspectRatioClass = `imageContainer${toTitleCase(
+		imageAspectRatio
+	)}`;
```

Comment:
Formatting isn't consistent between these two declarations, can we tidy up?

Reply:
I think this is a result of the linting/character limit per line rule, which spreads to the lines below if it's too long.